content/en/docs/reference/setup-tools/kubeadm/implementation-details.md

```diff
@@ -76,6 +76,20 @@ in a majority of cases, and the most intuitive location; other constant paths an
   - `front-proxy-ca.crt`, `front-proxy-ca.key` for the front proxy certificate authority
   - `front-proxy-client.crt`, `front-proxy-client.key` for the front proxy client
 
+## The kubeadm configuration API
```

Comment:
```suggestion
## The kubeadm configuration file format
```
We stretch the term “application programming interface“ really far. These are really configuration files that happen to align with our API conventions.